### ISSUE TITLE
Cannot read property 'Symbol(Symbol.iterator)' of undefined

### ISSUE THREAD

**Issue by minj** [mozilla/addons-validator#347]
Split from #217

Add-ons Validator (JS Edition) v0.0.7
npm v3.4.0
node v5.1.0
```
TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
    at Object.detectBadMozBindingURL (/usr/lib/node_modules/addons-validator/dist/webpack:/src/rules/css/detectBadMozBinding.js:8:50)
    at CSSScanner.processCode (/usr/lib/node_modules/addons-validator/dist/webpack:/src/scanners/css.js:46:28)
    at /usr/lib/node_modules/addons-validator/dist/webpack:/src/scanners/css.js:63:22
    at run (/usr/lib/node_modules/addons-validator/dist/webpack:/~/babel-core/~/core-js/modules/es6.promise.js:104:1)
    at /usr/lib/node_modules/addons-validator/dist/webpack:/~/babel-core/~/core-js/modules/es6.promise.js:115:1
    at flush (/usr/lib/node_modules/addons-validator/dist/webpack:/~/babel-core/~/core-js/modules/$.microtask.js:19:1)
    at doNTCallback0 (node.js:430:9)
    at process._tickCallback (node.js:359:13)
```

I only get this with [foxtrick](https://www.foxtrick.org/release/foxtrick-0.16.1.xpi) which is 5MB :/ 

I do have a few mini add-ons with a similar structure for testing various things but they don't produce this error.

**Comment by muffinresearch** [comment]
Great - thanks for the report!  We'll take a look and see if we can figure out what we're choking on.

**Comment by muffinresearch** [comment]
Ok the fix has landed. This will be in the next release which I expect should be tomorrow.

**Comment by minj** [comment]
@muffinresearch I can confirm that the fix prevents the exception. It was probably an exception caused by broken CSS. I've just fixed a couple of such problems thanks to the validator. :+1: 

Our CSS is not in the best of condition, I confess. Regardless, the CSS parser incorrectly throws `CSS_SYNTAX_ERROR` on CSS like this:
```js
#box /*.main{float:left;*/{width:auto !important;}
```
Seems you are looking too deeply into comments ;)

Obviously, this renders fine in browsers

**Comment by muffinresearch** [comment]
Yep I saw that - it's a separate bug that needs looking at - I think it might be a bug in the parser we're using becuase we already skip comments. I think that's being parsed as one rule for some reason - it might be because the comment is after the selector but before the `{`. I'll file a separate issue.

**Comment by minj** [comment]
```css
#box /*.main{float:left;}*/{width:auto !important;}
```
works fine, the key problem is unmatched braces
